apigw-sqs-lambda-sns/README.md

```diff
@@ -0,0 +1,141 @@
+# Webhook SNS Pattern - CDK Version
+
+This is a simplified AWS CDK implementation of a webhook integration pattern that receives webhook events via API Gateway, queues them in SQS, and processes them with Lambda to send SMS notifications via SNS.
+
+## Architecture
```

Comment:
The architecture explanation is brief and lacks details on how the components interact.
Expand the architecture explanation section to provide a more detailed description of how API Gateway, SQS, Lambda, and SNS work together in this pattern.

Adding an architecture diagram might be even better.